### PR TITLE
chore ~ add nvmrc

### LINKED ISSUE #1: Add `.nvmrc` for flexibility
Adding `.nvmrc` to this library will make it possible for contributors to use any version of `nodejs` they like, without a need to change their existing `nodejs` configurations.

### PR DESCRIPTION
**Pull request by murindwaz**
This PR will fix #1 ticket

Highlights 

- Add `.nvmrc` for clarity on which `nodejs` version to adopt while coding/building 
- Add README sections on How to use `.nvmrc`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,25 +2,41 @@
...
+- Navigate to the main folder and type command `npm run start:frontend` to start the front end. Navigate to `http://localhost:4200/`
+- Navigate to the main folder and type command `npm run start:backend` to start the back end. Navigate to `http://localhost:3333/songs` for the base path
 
-## To run the apps
+```shell
+git clone https://github.com/atbriggs/nsync
+cd nsync 
+nvm use
+npm i 
+npm run start:frontend
```
Comment (eharason) [inline README.md:22]: I think I prefer specifying that these are in different folders
```
Running the Frontend
- Navigate to the frontend folder
- `npm run start:frontend` to start the front end
- Server will be listening on http://localhost:4200/

Running the Backend
- Navigate to the backend folder
- `npm run start:backend` to start the back end
- Navigate to http://localhost:3333/songs for the base path 
```

Comment (eharason) [reply README.md:22]: or `cd frontend` / `cd backend` ... likely easier for students who want to get it installed and running ASAP lol

Comment (murindwaz) [reply README.md:22]: That is a great improvement. On the second review, don't you think `cd /frontend` would be misleading, given that both `npm run start:frontend` and `npm run start:backend` commands work when executed from the project root directory?

Comment (eharason) [reply README.md]: LOL good point @murindwaz! please disregard. that's how I interpreted it and that's how I've been running it 😅  but good point

**Review by eharason — APPROVED** [review]
Looks great!

**Pull request merged**